README.md

```diff
@@ -180,6 +180,19 @@ Requirements for building the project:
 * Java 17
 * Git
 * Python 3.11+ and PIP
+* Maven 3.8+
+
+Additionally, you will need to install into your Python env:
+* Black
+* Ruff
+* Build
+
+You can verify your requirements and run your first build like this:
+
+```bash
+mvn clean install -DskipTests
+mvn generate-sources
```

Comment:
we have maven wrapper so you don't need to install your own maven

you can run 
```
./mvnw ...
```

Reply:
generate-sources is a phase that is implied by "install", you don't need the second command